test/es_test_case.py

```diff
@@ -26,7 +26,7 @@ class ElasticsearchTestCase(DockerContainerTestCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        es_endpoint = cls._create_container('docker.elastic.co/elasticsearch/elasticsearch:5.5.3',
```

Comment:
It is worth mentioning that the Elasticsearch instance in AWS uses version `5.5.1` in its nodes.

Reply:
Interesting. But I still think we should attempt to match in the future even if we currently don't match them.